pkg/k8s/deployer_int_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build integration
+// +build integration
```

Comment:
note: so far from what ive reviewed only this int test has this `+build...` tag for backward compatibility, others dont. I guess not many people will use `<go1.17` in the current day

Reply:
Oh good point. I guess this was simply some copy&paste mistake. Will update it